src/convertToObject.js

```diff
@@ -3,16 +3,42 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl
+ *  in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfKeys = sourceString.split('\n');
+  const removeEmptyKeys = [];
+  const resultObject = {};
+
+  for (let i = 0; i < arrayOfKeys.length; i++) {
+    arrayOfKeys[i] = arrayOfKeys[i].replace(/^[^\w\d-]*/, '')
+      .replace(/[; ]+/g, ' ')
+      .replace(/:\s+/g, ':');
+
+    if (arrayOfKeys[i].length > 2) {
```

Comment:
You are checking if the length of the key is greater than 2, but keys can be one character long. You should check if the key is not an empty string instead.